Vue InstantSearch/geo-search/package.json

```diff
@@ -12,7 +12,7 @@
     "instantsearch.css": "7.3.1",
     "vue": "2.6.10",
     "vue-googlemaps": "0.1.2",
-    "vue-instantsearch": "3.8.1"
+    "vue-instantsearch": "latest"
```

Comment:
this shouldn't use latest, but update manually to the latest version. This way it won't break when we release a major version

Reply:
this change in the package.json needs to be reflected in yarn.lock

Reply:
Understood, good to know.
As breaking changes are expected to happen only on major version change, would it be relevant to use ^4.0.0 here, so  both upgrade patch and minor version updates will be pulled?

Reply:
yes

Reply:
I've done the change, is that good for you now?